### PR TITLE
Avoid sending backup start notification to HA Core 2021.12

### PR DESCRIPTION
**Pull request by agners**
## Proposed change

Avoid sending backup start notification to Home Assistant Core 2021.12. The implementation in 2021.12 has a bug when non-SQLite databases are used. The bug is scheduled to be fixed in 2022.2 (see https://github.com/home-assistant/core/pull/63847/).

## Type of change

- [ ] Dependency upgrade
- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (which adds functionality to the supervisor)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue:
- Link to documentation pull request:
- Link to cli pull request:

## Checklist

- [x] The code change is tested and works locally.
- [ ] Local tests pass. **Your PR cannot be merged unless tests pass**
- [ ] There is no commented out code in this PR.
- [ ] I have followed the [development checklist][dev-checklist]
- [ ] The code has been formatted using Black (`black --fast supervisor tests`)
- [ ] Tests have been added to verify that the new code works.

If API endpoints of add-on configuration are added/changed:

- [ ] Documentation added/updated for [developers.home-assistant.io][docs-repository]

[dev-checklist]: https://developers.home-assistant.io/docs/en/development_checklist.html
[docs-repository]: https://github.com/home-assistant/developers.home-assistant

### CODE REVIEW

File: supervisor/homeassistant/websocket.py [hunk 1]
```diff
@@ -21,8 +21,8 @@
 
 MIN_VERSION = {
     WSType.SUPERVISOR_EVENT: "2021.2.4",
-    WSType.BACKUP_START: "2021.12.0",
-    WSType.BACKUP_END: "2021.12.0",
+    WSType.BACKUP_START: "2022.2.0",
```
Comment (ludeeus) [inline supervisor/homeassistant/websocket.py:24]: Should this be `2022.1.0` so it includes dev/beta builds?

Comment (agners) [reply supervisor/homeassistant/websocket.py:24]: Yeah I was actually wondering: So `2022.2.0.dev20220110` < `2022.2.0`?

We bumped already to 2022.2.0, and fix will only be in a 2022.2.0 dev build...

Comment (ludeeus) [reply supervisor/homeassistant/websocket.py:24]: `2022.2.0.dev20220110` is considered lower than `2022.2.0`

Comment (agners) [reply supervisor/homeassistant/websocket.py]: Changed to `2022.1.0`.

**Review by ludeeus — APPROVED** [review]
LGTM 👍

**Pull request merged**